packages/suite/src/components/suite/Labeling/components/WalletLabeling.tsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import { TrezorDevice } from '@suite-types';
 import { useTranslation } from '@suite-hooks/useTranslation';
-
+import { useSelector } from '@suite-hooks/useSelector';
+import { selectLabelingDataForWallet } from '@suite-reducers/metadataReducer';
 interface WalletLabellingProps {
     device: TrezorDevice;
     shouldUseDeviceLabel?: boolean;
 }
 
 export const WalletLabeling = ({ device, shouldUseDeviceLabel }: WalletLabellingProps) => {
     const { translationString } = useTranslation();
+    const { walletLabel } = useSelector(state => selectLabelingDataForWallet(state, device.state));
 
-    let label: string | null = null;
-    if (device.metadata.status === 'enabled' && device.metadata.walletLabel) {
-        label = device.metadata.walletLabel;
+    let label: string | undefined = undefined;
+    if (device.metadata.status === 'enabled' && walletLabel) {
+        label = walletLabel;
```

Comment:
return `null` in case of `label` stays undefined? better than empty `span`

Reply:
ok 5a812239b2262b5c1005319e9d115364e4ef8e85